_data/simple-icons.json

```
@@ -17093,6 +17093,11 @@
             "hex": "000000",
             "source": "https://www.zebra.com"
         },
+        {
+            "title": "Zed Industries",
+            "hex": "084CCF",
+            "source": "https://zed.dev"
```

```suggestion
            "source": "https://github.com/zed-industries/zed/blob/ccc939124fa2f366b3029926447fd0a0c46a85c7/assets/icons/logo_96.svg"
```
Since this is the source of SVG, it should be changed from the webpage.

---

Right, I forgot that I didn't get the icon from the website! Thanks.